src/applications/appeals/shared/utils/upload.js

```diff
@@ -21,6 +25,9 @@ export const reMapErrorMessage = error => {
   return errormessageMaps?.[result] ?? error;
 };
 
+export const checkIsNameTooLong = (name = '') =>
```

Comment:
Would `checkIsFileNameTooLong` be a better name here? I read this quickly and thought it was a reuse of a name length check.

Reply:
Renamed!